### ISSUE TITLE
Support `.mobi` eBook format

### ISSUE THREAD

**Issue by myii** [TronFortyTwo/sturmreader#11]
Asked about `.mobi` on the main UBports Telegram group and the advice was to create an issue about it here.

Are there any plans to support the `.mobi` format in the future?

**Comment by TronFortyTwo** [comment]
Thanks for your report. Adding support for mobi files would be a good feature, I will consider adding it.

**Comment by TronFortyTwo** [comment]
If/when the calibre backends (in the calibre branch) lands, support for a lot of epub formats can be easily implemented hopefully

**Comment by myii** [comment]
@TronFortyTwo Great, sounds good.  Is it possible to test that right now by building using the `calibre` branch?

**Comment by TronFortyTwo** [comment]
@myii Yes, that is the way to test it. Keep in mind thought that the branch is experimental (I currently have problems with some encoding stuff)

**Comment by myii** [comment]
@TronFortyTwo First attempt at installing this via. `clickable` results in an error:

```bash
Package calibre:armhf is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'calibre:armhf' has no installation candidate
The command '/bin/sh -c apt-get update && apt-get install -y --force-yes --no-install-recommends gettext intltool:armhf qtdeclarative5-u1db1.0:armhf qtdeclarative5-usermetrics0.1:armhf qtdeclarative5-ubuntu-cont
ent1:armhf libpoppler-qt5-dev:armhf calibre:armhf python-apsw:armhf libmagickwand-6.q16-2:armhf liblqr-1-0:armhf && apt-get clean' returned a non-zero code: 100
Command exited with non-zero exit status 100, see above for details. This is most likely not a problem with Clickable.
```

**Comment by TronFortyTwo** [comment]
@myii Thanks for the report, that's because some confusion in Clickable selecting the right arch for calibre to install. I will eventually fix that. I've never seen that error before beacause I usually don't use clickable on my computer for the calibre branch.
That's my workflow: I access the ut device from USB and `clickable shell`
I create on the device a container with
`sudo debootstrap xenial container-name` and I enter with `sudo chroot container-name`
Then I install clickable inside the container and build Sturm Reader with `clickable --container-mode`
Hope it helps.
